integration-tests/tests/src/tests/sync/mixed.ts

```diff
@@ -107,46 +125,48 @@ function describeRoundtrip(typeName: string, value: Value, testValue: ValueTeste
   });
 }
 
-describe.skipIf(environment.missingServer, "mixed", () => {
-  importAppBefore("with-db");
+function performTests(flexibleSync: boolean) {
   authenticateUserBefore();
 
-  describeRoundtrip("null", null);
+  describeRoundtrip(flexibleSync, "null", null);
 
   // TODO: Provide an API to speficy storing this as an int
-  describeRoundtrip("int", 123);
+  describeRoundtrip(flexibleSync, "int", 123);
 
   // TODO: Provide an API to specify which of these to store
-  describeRoundtrip("float / double", 123.456);
+  describeRoundtrip(flexibleSync, "float / double", 123.456);
 
-  describeRoundtrip("bool (true)", true);
-  describeRoundtrip("bool (false)", false);
+  describeRoundtrip(flexibleSync, "bool (true)", true);
+  describeRoundtrip(flexibleSync, "bool (false)", false);
 
-  describeRoundtrip("string", "test-string");
+  describeRoundtrip(flexibleSync, "string", "test-string");
 
   // Unsupported:
   // describeSimpleRoundtrip("undefined", undefined);
 
   const buffer = new Uint8Array([4, 8, 12, 16]).buffer;
-  describeRoundtrip("data", buffer, (value: ArrayBuffer) => {
+  describeRoundtrip(flexibleSync, "data", buffer, (value: ArrayBuffer) => {
```

Comment:
I feel this is approaching the limits of positional arguments.

Reply:
Aha you might be right, I'll look at changing it